### DISCUSSION TITLE
Decide on name for QualityByDesign

### DISCUSSION THREAD

**Discussion by benhepworth** [camaraproject/SessionInsights#8] [General]
Related to Issue #3 
[QbD User Store PR](https://github.com/camaraproject/QualityByDesign/pull/7) for additional context

This is an extension of the original Quality Attenuation work, but not tied to a specific vendor. We chose the name Quality by Design with the idea that when an application is built, the application developer can design it in a way to ensure desired quality of experience.

On the TSC call some names were thrown out:
- Quality by Design (keep original name)
- Connectivity Insights Score
- Quality Action Score
- Quality Actions
- Quality Insights
- Application Quality Score
- Quality of Experience Insights

**Comment by benhepworth** [comment]
IMO the word Score shouldn't be in the name - it is much more than just providing a score

**Comment by Kevsy** [comment]
How about 'Quality Tracking'? 

'Tracking' implies an ongoing process, and just like tracking a parcel online, suggests the actions that can be taken.

The distinction would then be:

Connectivity Insights:  'if they were to start a session now, what quality is this end user likely to experience?'
Quality Tracking: 'for this ongoing session, what quality does the application perceive, and can the network do anything about that?'

**Reply by benhepworth** [reply]
My concern with the word "tracking" is that when put in the context of a network provider it could sound big brother-ish. What do you thinK?

BTW, I really like the distinctions you make. Well put.

**Reply by benhepworth** [reply]
what about Application Quality Insights?

**Reply by benhepworth** [reply]
I also like Quality of Experience Insights.

**Reply by Kevsy** [reply]
> My concern with the word "tracking" is that when put in the context of a network provider it could sound big brother-ish. What do you thinK?

Fair point! I hadn't thought of that...

> what about Application Quality Insights?

I think that could be misinterpreted as assessing the Application's role in quality, namely how well the Application is coded for networking (e.g. whether if follows guidance in [GSMA's 'Best Practice for 5G Developers'] (https://event-assets.gsma.com/Las-Vegas/Presentations/Ed-Lambert_ATT.pdf) or equivalent for fixed/Wi-Fi/Satellite)

> I also like Quality of Experience Insights.

Yes, I like that one. Or 'Session insights', in case there is another layer of 'Quality of Experience insights' which somehow includes end user feedback.

**Reply by benhepworth** [reply]
I like Session Insights...or what about Application Session Insights?

**Reply by Kevsy** [reply]
> I like Session Insights...or what about Application Session Insights?

I'd be fine with either of those - slight leaning to Session Insights as I think the 'Application' is implied by 'session'.

**Comment by benhepworth** [comment]
In general, what are everyone's thoughts about using Insights as the last word to align with Connectivity Insights naming?

**Comment by shafayetkhan** [comment]
Great ideas @benhepworth and @Kevsy!

I think adding Insights would fit well. At first, I was leaning towards Quality Insights since it pulls from both Quality on Demand and Connectivity Insights. But I also want to suggest a new one: Service Insights.

**Reply by benhepworth** [reply]
I'm not sure about using the word Service @shafayetkhan - in many cases it implies a service call by a technician and could add confusion.

**Reply by Kevsy** [reply]
@shafayetkhan re Service Insights: I see what you mean, but as I understand it, 'Service' is a popular term used in North America to indicate coverage - so it could be misinterpreted as relating to e.g. signal strength, 4G or 5G etc.

**Comment by benhepworth** [comment]
hi @hdamker - on the Connectivity Insights call this morning the decision was made to go with "Session Insights" for the name of the repo instead of Quality by Design. What needs to happen next in the process?

**Reply by hdamker** [reply]
> hi @hdamker - on the Connectivity Insights call this morning the decision was made to go with "Session Insights" for the name of the repo instead of Quality by Design. What needs to happen next in the process?

@benhepworth next step would be to create an issue with the request to rename (and some rational) within https://github.com/camaraproject/APIBacklog, including a PR on the original API Proposal. In addition it would be good to have the User Story done and merged (#7) and some minutes about the call this morning (I assume that the other Maintainers of ConnectivityInsights were involved in the decision). Technically changing the name is afterwards a small admin task.

**Comment by benhepworth** [comment]
New name has been decided upon: Session Insights
Link to meeting notes where this decision was made: https://lf-camaraproject.atlassian.net/wiki/x/mIDoAQ
Link to Issue in API Backlog to rename Quality by Design to Session Insights: https://github.com/camaraproject/APIBacklog/issues/116
Link to PR in API Backlog to update original docs and to resolve Issue #116 in API Backlog: https://github.com/camaraproject/APIBacklog/pull/117

@hdamker - ok, all of this has been done. The only outstanding thing now is to merge the two current open issues in the QualityByDesign repo (README updates and User Story). Upon approval from @Kevsy or @maheshc01 I will merge these (should have this hopefully this week). Then the rename can happen via the open Issue/PR in API Backlog. Thanks for your direction in this.
